source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -33,12 +33,21 @@ You can create a Prometheus service within your PaaS space and allow it to bind
   - `cf create-service gds-prometheus prometheus <service-instance-name>`
 2. Either [update your app's manifest.yml] to bind your new service or bind it using the CLI
   - `cf bind-service <app-name> <service-instance-name>`
+3. If your app uses a custom domain, you need to make sure that the [authorization header and other app headers][] are being forwarded otherwise the basic auth to the metrics endpoint will fail and your app won't have access to your other headers.
+  - `cf update-service <cdn-route-service> \
+    -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization"]}'`
```

Comment:
I guess my worry is that a user will simply copy/paste this command line, and might miss that they also need to forward, say, "Cookie" in their particular app's case.  I don't think the words "and other app headers" in the copy above are enough to prevent this.

Maybe:

 - `cf update-service <cdn-route-service> \
    -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization", <any-other-headers-needed-by-your-app>]}'`

This has the benefit that the command line can't be simply copy-pasted; the user has to make an active choice to remove the "any other headers" bit.

Reply:
👍